image.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Assignment-4</title>
+</head>
+<body>
+  <div class="container">
+    <div>
+      <a href="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/Adobe_Corporate_Logo.png/1200px-Adobe_Corporate_Logo.png"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/Adobe_Corporate_Logo.png/1200px-Adobe_Corporate_Logo.png" width="628px" height="124px" alt="Unsplash-Wallpaper" align="right"></a>
```

Comment:
u have used inline styles for img tag which are not applied to the image please correct it